config/crd/kustomizeconfig.yaml

```diff
@@ -0,0 +1,19 @@
+# This file is for teaching kustomize how to substitute name and namespace reference in CRD
```

Comment:
I'm not sure what this does

Reply:
The config directory is mostly generated here :)